src/components/Kebab.tsx

```diff
@@ -13,14 +13,15 @@ const KebabButton = styled.button`
 `;
 
 function Kebab() {
-  const [isKebab, setIsKebab] = useState(false);
-  const optionsRef = useRef < HTMLDivElement > null;
+  const [isKebab, setIsKebab] = useState<boolean>(false);
+  const optionsRef = useRef<HTMLDivElement>(null);
 
   const handleKebabOpen = () => {
     setIsKebab((prev) => !prev);
   };
 
-  const handleKebabClose = (e) => {
+  const handleKebabClose = (e: any) => {
```

Comment:
아래에서 버튼에 들어가는 핸들러여서 버튼 엘리먼트를 React.FocusEvent의 제네릭으로 넣어주면 될것같습니다!
```suggestion
  const handleKebabClose = (e:  React.FocusEvent<HTMLButtonElement>) => {
```